content/news/2020-09-28-wallpapers-contest.md

```diff
@@ -0,0 +1,34 @@
++++
+type = "news"
+title = "Call for Wallpapers"
+author = "bitigchi"
+date = "2020-10-01 13:00:00+03:00"
+tags = ["wallpapers", "contest"]
++++
+
+One of the most common feedback that we receive is that the Haiku needs some nice colourful wallpapers shipping with Haiku releases. Suffice to say, that we agree with this completely. We definitely want to improve the current situation. And we need your help.
+
+Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned on the screen on an appropriate position according to the screen resolution. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
```

Comment:
I think the positioning of the logo isn't even resolution dependend...

```suggestion
Haiku currently ships with only one (1) wallpaper, and it's not a complete wallpaper per se, it's just the Haiku logo positioned at fixed coordinates on the screen. To be fair, there is a reason we only use the Haiku logo as the default wallpaper; we want Haiku to be usable on systems that have memory as low as 256 MB, therefore each byte that we can spare counts.
```

Reply:
I have noticed that the logo position is different on different systems. Sometimes it's on the centre, sometimes it's close to the bottom. How does it work?

Reply:
Are you sure those systems ran the same resolution? AFAIK, the logo appears at fixed X/Y coordinates, origin being the top left corner of the screen. The higher the resolution the further the logo moves up and left.